spring-greeter/pom.xml

```diff
@@ -66,6 +66,14 @@
             <scope>provided</scope>
         </dependency>
 
+	<!-- Import the javax.servlet-api dependency -->
+	<dependency>
+		<groupId>javax.servlet</groupId>
+		<artifactId>javax.servlet-api</artifactId>
+		<version>4.0.1</version>
```

Comment:
An individual quickstart should not be declaring the version of the dependency, instead for something like the servlet-api the version should be omitted so the version defined in the bom references from the parent project will be used.

Reply:
it should use instead our fork of the servlet specs, i.e

```
        <dependency>
            <groupId>org.jboss.spec.javax.servlet</groupId>
            <artifactId>jboss-servlet-api_4.0_spec</artifactId>
            <scope>provided</scope>
        </dependency>
```

Reply:
Added the changes and now it's working fine !